### DISCUSSION TITLE
Upgrade option in Resolution Rule

### DISCUSSION THREAD

**Discussion by sukhpalp** [itsToggle/plex_debrid#582] [Q&A]
Could somebody please explain what the Upgrade option in the Resolution rules does?

**Comment by ChrisFiola** [comment]
Upgrade will scrape your media and if your media is in a lower resolution dans the upgrade value it will try to upgrade if it finds an upgraded resolution through your sources.

Example: 
1 - In your version you may have set a rule that will grab any sources with a resolution of 1080p
2 - Now you want to upgrade your sources to a higher resolution than 1080p if ever there is one available. For that reason you will add the rule: "upgrade >=1081" to your version.

If you have media in 1080p it will look for sources that have a resolution equal or higher than 1081p and if it finds one it will add it to your debrid. I have found that if I set "upgrade >=1080", it will not try to find upgrades higher than 1080p because according to the rule "greater or equal" it is already equal to 1080 so there is no need to upgrade. For that reason I have had good results by setting the upgrade value +1 of the value I want to upgrade from.
